src/qt/treectrl.cpp

```diff
@@ -342,15 +1234,64 @@ wxTreeItemId wxTreeCtrl::DoInsertAfter(const wxTreeItemId& parent,
                                    int image, int selImage,
                                    wxTreeItemData *data)
 {
-    return wxTreeItemId();
+    wxCHECK_MSG(parent.IsOk(), wxTreeItemId(), "invalid tree item");
+    wxCHECK_MSG(idPrevious.IsOk(), wxTreeItemId(), "invalid tree item");
+
+    const QTreeWidgetItem *qTreeItem = wxQtConvertTreeItem(parent);
+    const int index = qTreeItem->indexOfChild(wxQtConvertTreeItem(idPrevious));
+    return DoInsertItem(parent, index + 1, text, image, selImage, data);
 }
 
 wxTreeItemId wxTreeCtrl::DoTreeHitTest(const wxPoint& point, int& flags) const
 {
-    return wxTreeItemId();
+    int w, h;
+    GetSize(&w, &h);
+    flags = 0;
+    if ( point.x < 0 ) flags |= wxTREE_HITTEST_TOLEFT;
```

Comment:
Please don't put if statement body on the same line.

Reply:
Fixed.